src/app/fdctl/run/run.c

```diff
@@ -649,7 +649,7 @@ check_configure( config_t * const config ) {
                  "to create the mounts correctly. This must be done after every system restart before running "
                  "Firedancer.", check.message ));
 
-  check = ethtool.check( config );
+  check = ethtool_channels.check( config );
   if( FD_UNLIKELY( check.result!=CONFIGURE_OK ) )
```

Comment:
Please add another check here for `ethtool_gro` as well

Reply:
Addressed